### PR TITLE
Add logo client manager

### PR DESCRIPTION
**Pull request by GusBaamonde**
**Logo client manager for small resolutions**

header-client-manager2

### CODE REVIEW

**Comment by andresmoschini** [comment]
Creo que el logo de Doppler no debería ir, no?

**Comment by GusBaamonde** [comment]
> Creo que el logo de Doppler no debería ir, no?

Si si se puede sacar lo deje para calcular, pero si, si lo sacamos no afecta en nada

File: assets/scss/templates/_header.scss [hunk 1]
```diff
@@ -21,9 +21,40 @@
...
+}
+
+.dp-header--cm {
+  .dp-logo--cm {
+    margin-bottom: 15px;
+    max-width: 180px;
+    max-height: 50px;
+    width: 180px;
+    height: 50px;
+    background: #CCC;
```
Comment (cbernat) [inline assets/scss/templates/_header.scss]: Please check here, to use color variables.

File: assets/scss/templates/_header.scss [hunk 2]
```diff
@@ -21,9 +21,40 @@
...
+    &.dp-header--cm{
+      min-height: 185px;
+      overflow: visible;
+    }
+  }
+}
+
+.dp-header--cm {
+  .dp-logo--cm {
+    margin-bottom: 15px;
```
Comment (andresmoschini) [inline assets/scss/templates/_header.scss]: level?

File: assets/scss/templates/_header.scss [hunk 3]
```diff
@@ -21,9 +21,40 @@
...
+    max-width: 180px;
+    max-height: 50px;
+    width: 180px;
+    height: 50px;
+    background: #CCC;
+    display: block;
+    margin-left: 3%;
```
Comment (andresmoschini) [inline assets/scss/templates/_header.scss]: is it fine?

File: assets/scss/templates/_header.scss [hunk 4]
```diff
@@ -21,9 +21,40 @@
...
+    margin-top: 15px;
```
Comment (andresmoschini) [inline assets/scss/templates/_header.scss]: level?

File: assets/scss/templates/_header.scss [hunk 5]
```diff
@@ -21,9 +21,40 @@
...
+    }
+  }
+}
+
+.dp-header--cm {
+  .dp-logo--cm {
+    margin-bottom: 15px;
+    max-width: 180px;
+    max-height: 50px;
+    width: 180px;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_header.scss]: need width if uses max-width?

File: assets/scss/templates/_header.scss [hunk 6]
```diff
@@ -21,9 +21,40 @@
...
+
+    &.dp-header--cm{
+      min-height: 185px;
+      overflow: visible;
+    }
+  }
+}
+
+.dp-header--cm {
+  .dp-logo--cm {
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_header.scss]: Remember add vertical align for logos with less height than 50px

File: assets/scss/templates/_header.scss [hunk 7]
```diff
@@ -21,9 +21,37 @@
...
+    display: none;
   }
 }
 
+.dp-logo--cm {
+  display: none;
+}
+
 .header-wrapper {
   padding-left: 2%;
```
Comment (IgnacioRodrigues) [inline assets/scss/templates/_header.scss]: Maybe you can use variables?

File: assets/scss/templates/_header.scss [hunk 8]
```diff
@@ -382,6 +426,7 @@ a {
       font-size: 14px;
```
Comment (IgnacioRodrigues) [inline assets/scss/templates/_header.scss]: Maybe you can use variable in this case?

File: assets/templates/header.html [hunk 9]
```diff
@@ -46,7 +46,8 @@
     <li class="nav-item"><a href="forgot-password.html" class="nav-link">Forgot</a></li>
   </ul><!-- nav -->
 </section><!-- section-module -->
-    <header class="header-main header-open">
+    <header class="header-main header-open dp-header--cm">
+      <div class="dp-logo--cm"><img src="https://via.placeholder.com/180x50"></div>
```
Comment (cbernat) [inline assets/templates/header.html]: Why are you using this placeholder? --> "https://via.placeholder.com/180x50
I think we have one of our own in doppler.

**Comment by cbernat** [comment]
Great! 🚀

**Comment by damianmuti** [comment]
Good job, @GusBaamonde !
If we are going to need a few other image placeholders, I guess it could be a good idea to have our own internal service to get images _on-the-fly_ through url `GET` parameters, something like `https://services.fromdoppler.com/images?w=[width]&h=[height]&b=[background-color]`

I can toy around that when I get some spare time. 💪

**Comment by GusBaamonde** [comment]
> Good job, @GusBaamonde !
> If we are going to need a few other image placeholders, I guess it could be a good idea to have our own internal service to get images _on-the-fly_ through url `GET` parameters, something like `https://services.fromdoppler.com/images?w=[width]&h=[height]&b=[background-color]`
> 
> I can toy around that when I get some spare time. 💪

Excellent! If we need to make a component for placeholders, let's talk about this! Thanks @damianmuti

**Pull request merged**